packages/rundotnetdll.vm/rundotnetdll.vm.nuspec

```
@@ -2,12 +2,13 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
   <metadata>
     <id>rundotnetdll.vm</id>
-    <version>2.2.0.20250219</version>
-    <description>A simple utility to list all methods of a given .NET Assembly and to invoke them.</description>
+    <version>2.2.0.20250505</version>
+    <description>RunDotNetDll is a simple utility to list all methods of a given .NET Assembly and to invoke them.</description>
```

[nitpick] Remove unrelevant details
```suggestion
    <description>RunDotNetDll is a utility to list all methods of a given .NET Assembly and to invoke them.</description>
```